cookiecutter-django-app/{{cookiecutter.repo_name}}/requirements/constraints.txt-E

```diff
@@ -0,0 +1,12 @@
+# Version constraints for pip-installation.
```

Comment:
Please verify if the file name was miswritten.

Reply:
This was a temp file added by mistake